third-party/CMakeLists.txt

```diff
@@ -119,8 +119,13 @@ set(LIBTERMKEY_URL http://www.leonerd.org.uk/code/libtermkey/libtermkey-0.20.tar
 set(LIBTERMKEY_SHA256 6c0d87c94ab9915e76ecd313baec08dedf3bd56de83743d9aa923a081935d2f5)
 endif()
 
+if(MSVC)
+set(LIBVTERM_URL https://github.com/b-r-o-c-k/libvterm/archive/eb386b1d82f7d07363c9133b7aa06902ccd555fe.tar.gz)
```

Comment:
Please try to work with upstream on MSVC support.  I thought VS 2015 had C99 support, so why don't VLA work?

Reply:
Although they have added support for some C99 features, they still don't support VLAs. The C90 standard is the only one they aim to be fully compliant with. You can read their response to the VLA feature request [here](https://visualstudio.uservoice.com/forums/121579-visual-studio-ide/suggestions/11580261-support-c99-variable-length-arrays).